src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,79 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final String COMMA = ",";
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String RESULT = "result";
+    private int supplyAmount = 0;
+    private int buyAmount = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+
+        supplyAmount = 0;
+        buyAmount = 0;
+        File fromFile = new File(fromFileName);
+        final File toFile = new File(toFileName);
```

Comment:
There's no need to declare `toFile` as final since it's not used in any inner classes or lambda expressions. You can remove the `final` modifier.